package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "(trap 'kill 0' SIGINT; docker compose up redis & next dev)",
```

Comment:
This bash script is essentially: "start `redis` and `next dev`, have them both output to the console, and when I hit Cmd-C, quit both"

See: https://spin.atomicobject.com/2017/08/24/start-stop-bash-background-process/

Why do it this way?  There's no change to the developer experience at all, you can still start your dev server locally with `npm run dev`.  No need to make it a two-step process.

Reply:
I am a strong believer in [makefiles > npm scripts](https://medium.com/finn-no/makefiles-for-frontend-1779be46461b), but probably a conversation for another day and a tech debt task 😉